src/components/DataTable/utils/renderHelpers.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Chip} from '~/index';
+
+export type LocaleOptions = {
```

Comment:
As we change how the functions work, I guess we can also change the typing by `localeOptions = Intl.NumberFormatOptions | Intl.DateTimeFormatOptions`